### PR TITLE
  Implement a PATCH for updating Name and CompanyName for Client && Implement update and soft delete post project for client && Somenone visited your profile notification

### PR DESCRIPTION
**Pull request by KAJOiq**

### CODE REVIEW

**Review by diyar214 — APPROVED** [review]
LGTM!

File: Controllers/Mobile/v1/ClientsController.cs [hunk 1]
```diff
@@ -17,8 +17,38 @@ namespace AonFreelancing.Controllers.Mobile.v1
...
     public class ClientsController(MainAppContext mainAppContext,
                                     ActivitiesService activitiesService,
-                                    UserService userService) : BaseController
+                                    UserService userService,
+                                    UserManager<User> userManager, AuthService authService, RoleManager<ApplicationRole> roleManager) : BaseController
     {
+ 
+
+
+        // before your review, you should i have skill issue here cuz the time is now 11:56 PM :)
```
Comment (Yousif-Alsheikh) [inline Controllers/Mobile/v1/ClientsController.cs:26]: clean code and English are crying together in the corner now. 👀

**Review by Yousif-Alsheikh — APPROVED** [review]
we will do another check for soft delete.

**Review by Yousif-Alsheikh — APPROVED** [review]

File: Controllers/Mobile/v1/ClientsController.cs (hunk 1 above)
Comment (KAJOiq) [reply Controllers/Mobile/v1/ClientsController.cs:26]: you should i have ?
i don't know how i wrote this :)

**Comment by KAJOiq** [comment]
Project IsDeleted may be make some bugs in our endpoints, so we will check my pr togother.
@diyar214 @Yousif-Alsheikh @Humam-Yousif @MohammedLilo

**Review by Yousif-Alsheikh — COMMENTED** [review]
need to be checked

**Review by MohammedLilo — APPROVED** [review]
approved after fixes

**Pull request merged**